content/en/tracing/guide/configuring-primary-operation.md

```diff
@@ -82,15 +82,123 @@ By default, the resource name would be set to this as it’s the name of the fun
 
 ## OpenTracing
 
-When using Datadog, the Opentracing operation name is a resource and the Opentracing "component" tag is Datadog's span name. To define (in Opentracing terms) a span that has the resource "/user/profile", and the span name "http.request", use this Go example:
+When using Datadog, the OpenTracing operation name is a resource and the Opentracing "component" tag is Datadog's span name. For example, to define (in OpenTracing terms) a span that has the resource "/user/profile", and the span name "http.request":
```

Comment:
```suggestion
When using Datadog, the OpenTracing operation name is a resource and the OpenTracing "component" tag is Datadog's span name. For example, to define (in OpenTracing terms) a span that has the resource "/user/profile", and the span name "http.request":
```

Reply:
Good catch, I thought I capitalized all the `T`s!